src/Tool.java

```diff
@@ -0,0 +1,33 @@
+
+import data.*;
+import engine.*;
+import errors.*;
+import utils.SortType;
+import java.io.File;
+
+public class Tool {
+    private final Config config;
+
+    public Tool(Config cfg) {
+        this.config = cfg;
+    }
+
+    public void launch() throws FileCommandException, NoDataException, NoSortException {
```

Comment:
Incorrect formatting in the method. 
Simple way how to format code: 
Select content of the file using hot key `command + A`
And then use hot key `option + command + L`